src/components/debug/PairingDashboard.tsx

```diff
@@ -19,33 +19,34 @@ import {
 } from "@chakra-ui/react";
 import { SpinnerIcon } from "@chakra-ui/icons";
 import api from "../../services/api";
-import { InterViewPoolStatus } from "../../types";
+import { InterviewPoolStatus } from "../../types";
+import { getInterviewPoolStatus } from "../../services/interview";
+import { devPrint } from "../utils/RandomUtils";
 
 const PairingDashboard = () => {
   const [loading, setLoading] = useState(false);
   const [signal, setSignal] = useState(false);
   const [response, setResponse] = useState("");
 
-  const [signupData, setSignUpData] = useState<InterViewPoolStatus>({
+  const [signupData, setSignUpData] = useState<InterviewPoolStatus>({
+    numberSignUp: 0,
```

Comment:
You don't need to set this to 0 as you're already checking below if it's undefined and returning 0 if it is.